lib/orm.php

```diff
@@ -2196,6 +2196,59 @@ public function save() {
 		return $success;
 	}
 
+	/**
+	 * Inserts multiple rows in a single query. Expects new rows as it's a strictly insert function, not an update one.
+	 *
+	 * @param array $models Array of model instances to be inserted.
+	 *
+	 * @return bool True for successful insert, false for failed.
+	 *
+	 * @example From the Indexable_Link_Builder class: $this->seo_links_repository->query()->save_many( $links );
+	 *
+	 * @throws \Exception Instance to be inserted is not a new one.
+	 */
+	public function insert_many( $models ) {
+		$values  = [];
+		$success = true;
+
+		foreach ( $models as $model ) {
```

Comment:
Validate that every instance of `$models` is of the same class.

Reply:
Decided with the team to move that validation into a future task so that we can properly enhance the function to be able to proceed with the process even if it receives instances of different classes.